### PR TITLE
fix(searchbox): add aria-hidden to svg icons

### PR DESCRIPTION
**Pull request by pat-sullivan**
**Summary**
Added `aria-hidden="true"` attributes to the following SVG icons used in the repo: 
- ais-SearchBox-resetIcon
- ais-SearchBox-loadingIcon
- ais-SearchBox-submitIcon

This is in response to a Shopify customer support ticket saying these icons were flagged (for missing title attributes) during an accessibility audit. Since these icons are nested below buttons with title attributes, an aria-hidden attribute seemed the better option.

### CODE REVIEW

File: packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ describe('searchBox', () => {
           height="10"
           viewBox="0 0 40 40"
           width="10"
+          aria-hidden="true"
```
Comment (Haroenv) [inline packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx:68]: is there a forgotten commit? I can see all the test updates, but the code didn't change 🤔

**Review by Haroenv — CHANGES_REQUESTED** [review]
Thanks @pat-sullivan, this looks like a great start! we should indeed have these images aria-hidden, as they are purely visual and have a parent with a title

File: packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx (hunk 1 above)
Comment (Haroenv) [reply packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx:68]: react-instantsearch-hooks-web/ui is updated, but not React InstantSearch dom / components, InstantSearch.js / components, Vue InstantSearch / components :)

Comment (pat-sullivan) [reply packages/instantsearch.js/src/widgets/search-box/__tests__/search-box.test.tsx:68]: You're right 😅 I did a search for exact match classnames, but it looks like some are dynamically built. Thanks for catching. Will update soon.

File: packages/vue-instantsearch/src/components/__tests__/__snapshots__/RefinementList.js.snap [hunk 2]
```diff
@@ -46,6 +47,7 @@ exports[`allows search bar classes override when it's searchable 1`] = `
              role="img"
              viewbox="0 0 20 20"
              width="1em"
+             aria-hidden="true"
```
Comment (Haroenv) [inline packages/vue-instantsearch/src/components/__tests__/__snapshots__/RefinementList.js.snap:50]: these Vue snapshots need aria-hidden as the first property (it's alphabetically sorted)

Comment (pat-sullivan) [reply packages/vue-instantsearch/src/components/__tests__/__snapshots__/RefinementList.js.snap]: @Haroenv Thanks for pointing that out! I've made the necessary adjustments. However I can't seem to get past that flaky unit test on `packages/react-instantsearch-hooks-server/src/__tests__/modules-none.test.tsx`

**Review by Haroenv — APPROVED** [review]
thanks! This looks good to me

**Pull request merged**